README.md

```diff
@@ -1,5 +1,6 @@
 # OSPO Project Template
 
+![Release](https://img.shields.io/badge/OSS%20Template-0.3.1-brightgreen.svg)
```

Comment:
Do we think we will need/want a programatic way to crawl repos for updates? this is great for visual awareness, but less for automated.

Reply:
This was the one thing I was worried about when I suggested adding a "version" badge to the README. I know there _has_ to be a way to do so, and maybe there's even a Github action we could leverage here.

That said, for _right now_ manual updates are probably fine, but IMO this warrants a follow-up ticket to look into creating a proper CI pipeline for templates.

Reply:
What I wanted to avoid was having a dynamic badge as that would defeat the purpose. It's meant to represent a fixed point in time when the template was used. That said, if we can maintain that _and_ not have to update this link manually, that's clearly better. Agreed on this being a follow up improvement.

Reply:
Yeah, I think the improvement here is more reducing the toil / the probability that we forget to do incremental bumps anywhere the template version is referenced across the repository in the future. Given that we could also leverage a CI workflow to introduce markdown linting on all the template content, it seems worth prioritizing, but is certainly not a blocker for this PR.